src/main/resources/htmlpublisher/HtmlPublisherTarget/BaseHTMLAction/index.groovy

```diff
@@ -26,9 +28,10 @@ def serveWrapper() {
     def footer = HtmlPublisher.class.getResourceAsStream("/htmlpublisher/HtmlPublisher/footer.html").text
 
     raw(header)
+    script(src: "${Jenkins.get().getRootUrl() + Functions.getResourcePath()}/plugin/htmlpublisher/js/htmlpublisher.js", type: "text/javascript")
```

Comment:
Here I also need `resURL`, see https://github.com/jenkinsci/htmlpublisher-plugin/pull/312/files#r1824514341 for context. In this case it might be solvable if I call `Functions.initPageVariables`. It takes `JellyContext` parameter though, and we're in a groovy view, so I'm not sure if it's going to work. Haven't tried it locally yet but will give it a shot.

Reply:
Stapler's implementation of Groovy views makes reference to `JellyContext`, so it might work?

Reply:
Yeah, it just works. I'm confused by how I had access to `rootURL` (https://github.com/jenkinsci/htmlpublisher-plugin/pull/312/files#diff-d72667a208d99a60782dc45554c51942d712562357e76717b9ae37975ad7b553R52) without calling `initPageVariables`.